circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Maybe one other fallback before directly selecting `/Applications/Xcode.app` (which isn't necessary the one we want) is to `ls -1 /Applications | grep '^Xcode*.app'`, then use `plutil` to check the `CFBundleIdentifier` and extract the `CFBundleVersion` ourselves. We be a little like re-creating `mdls` manually but shouldn't be too much work and probably more reliable.